tests/conftest.py

```diff
@@ -67,16 +31,87 @@ def input_dir(tmp_path):
         pass
     with open(f'{input_dir}/test_01.jpg', 'w'):
         pass
-    with open(f'{input_dir}/metadata.csv', 'w') as csvfile:
-        writer = csv.writer(csvfile)
-        writer.writerow(['uri'] + ['title'] + ['file_identifier'])
-        writer.writerow(['/repo/0/ao/123'] + ['Test Item'] + ['test'])
-        writer.writerow(['/repo/0/ao/456'] + ['Tast Item'] + ['tast'])
     return str(f'{input_dir}/')
 
 
+@pytest.fixture()
+def aspace_delimited_csv():
+    with open('tests/files/aspace_metadata_delimited.csv') as f:
+        reader = csv.DictReader(f)
+        yield reader
+
+
+@pytest.fixture()
```

Comment:
I think this fixture can be removed? And any others that aren't being used.

Reply:
I will also handle this in the `reconcile` refactor